docs/src/modules/javascript/pages/value-entity-videos.adoc

```diff
@@ -1,12 +1,10 @@
 = Value entities and stateful services: Video 1-2-3
 
-The following videos will give you a quick overview of building a start service with value entities.
+These videos will give you a quick overview of implementing Value Entities in Akka Serverless. In less than five minutes, you can build a stateful serverless service using Value Entities. These videos walk you through the three steps to you need to develop:
```

Comment:
@retgits, don't you think it will be confusing that the steps in these videos are different than what is in the doc? In the doc, we suggest they design first and choose their state model. And, your second video isn't really about "choosing" the state model, I don't remember you even mentioning Event Sourced entities? It is about implementing in a .js file.

Reply:
Sorry, I thought you were just going through possibilities. When I looked at the tutorials there was no interstitial location where the videos seemed a natural fit. And the new page added after the tutorials felt a bit disarticulated, outside the flow. Whereas, the vid page seemed like an enhancement for the value entity material. But for ease of maintenance, I can see that this would be better off on the Doc Repo side. Let me know, I can move it in a trice.
Oh, BTW, Peter helped me out, and I now know how to build on the SDK side. Right now it uses the default Antora UI, but I think we can work to update that. At least we can validate now.

Reply:
Great that you can build! What about in the Tutorial section before the IOT and shopping cart? It is a Hello World example in JavaScript, that's how I would name it.

Reply:
Ok Great idea - I'll rebrand it and put it there as a 'Hello World'